src/lib/index.js

```diff
@@ -451,8 +451,12 @@ lib.addStyleRule = function(selector, styleString) {
     else lib.warn('addStyleRule failed');
 };
 
-lib.getTranslate = function(element) {
+// TODO (alexcjohnson): the following translate/scale functions should
+// get moved to components/drawing rather than the generic lib.
```

Comment:
done in https://github.com/plotly/plotly.js/pull/1332/commits/66baf516524d114ba2044b2f432a3c3441c5ea43

Any final comments @etpinard ?